### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by burnash**
This PR adds a first draft of GitHub issues templates derived from GitHub generated templates.

Further improvement could be to implement [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (rudolfix) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: please look at https://github.com/dlt-hub/verified-sources/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

we should get more context on a bug (not always applicable) ie.
- data source type (verified source? API? just data)
- destination (bigquery? etc.)
- runtime env. so we know this is production or local laptop or Colab notebook

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (rudolfix) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: Before asking questions below I'd ask about context in which dlt is used: ie. we could ask
- I'm already dlt user
- I'd consider to use dlt but I'm lacking a feature
- I run dlt in production
- I run it for fun
etc.

**Review by rudolfix — CHANGES_REQUESTED** [review]
check out the suggestions. let's add PR template that asks people to link issue

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (rudolfix) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: ohhh I'd also ask them to go to slack before and report there. I think we should have people check out tech help on slack before reporting bugs

**Comment by burnash** [comment]
@rudolfix please review. You can see the rendered version in [my branch](https://github.com/dlt-hub/dlt/blob/enh/github_issues_template/.github/ISSUE_TEMPLATE/bug_report.yml).

**Comment by burnash** [comment]
@rudolfix added a PR template as well, please take a look

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+description: Report a problem with dlt
+labels: ["kind:bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting a bug for dlt! Please fill out the sections below.
+        If you are not sure if this is a bug or not, please join our [Slack](https://join.slack.com/t/dlthub-community/shared_invite/zt-1slox199h-HAE7EQoXmstkP_bTqal65g)
+        and ask in the #3-technical-help channel.
+  - type: dropdown
```
Comment (rudolfix) [inline .github/ISSUE_TEMPLATE/bug_report.yml:12]: should be it a normal input? so people just paste the version

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+    attributes:
+      label: dlt version
+      description: What dlt version are you using?
```
Comment (rudolfix) [inline .github/ISSUE_TEMPLATE/bug_report.yml:15]: we could maybe tell them to do `dlt --version` to get it

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    attributes:
+      label: Python version
+      description: What Python version are you using?
+      multiple: false
+      options:
+        - "3.11"
+        - "3.10"
+        - "3.9"
+        - "3.8"
+        - "Below 3.8"
```
Comment (rudolfix) [inline .github/ISSUE_TEMPLATE/bug_report.yml:93]: we do not support <3.8

**Review by rudolfix — CHANGES_REQUESTED** [review]
just a few suggestions. then we can merge. I wonder what happens when issue is created using form. should still be in Markdown format I just wonder how it looks like :)

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (burnash) [reply .github/ISSUE_TEMPLATE/bug_report.yml:12]: Good point. Let's change that, also reduces the maintenance

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 5 above)
Comment (burnash) [reply .github/ISSUE_TEMPLATE/bug_report.yml:93]: I've intentionally put this as a red flag, even though dlt won't install on Python<3.8. If you think we don't need it I can remove that option.

**Review by rudolfix — APPROVED** [review]

**Pull request merged**